.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy dspace
+
+on:
+  push:
+
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: dspace-dep-1
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Ready script
+      - name: Ready script
+        run: chmod u+x $GITHUB_WORKSPACE/build-scripts/deploy-scripts/deploy.sh
```

Comment:
`+x` change in git
https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows
git update-index --chmod=+x foo.sh